specifications/secure-launch-specification.rst

```diff
@@ -713,6 +747,84 @@ Using this logic, the resulting operation to measure the policy would be as:
 The result, `M_policy`, will be a hash of the policy that can then be extended
 into one, or more if using as a cap value, PCR(s).
 
+Note that current SLRT specification version doesn't require measuring the
+policy, neither does it have appropriate policy entry type for that measurement.
+
+Measuring the SLRT
+------------------
+
+In revision one of the SLRT, the only table that needs to be measured is the
```

Comment:
If there is a need to measure the SLRT, the recommendation is that the vendor info table, i.e. one of `SLR_ENTRY_INTEL_INFO`, `SLR_ENTRY_AMD_INFO` or `SLR_ENTRY_ARM_INFO`, is the only one that should be measured. The remainder of the SLRT is meta-data, ......